services/QuillLMS/config/routes.rb

```diff
@@ -364,6 +364,8 @@
     namespace :v1 do
       get 'activities/uids_and_flags' => 'activities#uids_and_flags'
       get 'rule_feedback_histories' => 'rule_feedback_histories#by_conjunction'
+      get 'rule_feedback_history/:rule_uid' => 'rule_feedback_histories#rule_detail'
```

Comment:
Do we need to also pass the activity id here? Aren't some rules shared across activities? cc @anathomical?

Reply:
So, technically, `Rule` records are tied to `Prompt` records via the `PromptsRule` model.  It's possible that you want to be able to both get all `FeedbackHistory` for a specific rule, AND get only feedback for that rule if it belongs to a specific `Activity`.  I don't remember precisely what the UX requires here.

Reply:
I just re-scanned the project spec, but don't see a clear requirement to filter on Activity here. How about we move forward and add this modification in V2 if it's ever requested?

Reply:
This page is nested below the activity, and you can only get to it by clicking on the rule from the rules analysis page that is explicitly activity-specific. In the mockup for the interface, you can see that the base prompt for the responses is shown at the top of the table, which doesn't make sense if there are responses being shown from more than one activity. So I do think it's clear from the spec that this is activity specific. That said, I'm fine moving on for now if you want to address it later. I don't know how many rules are cross-activity anyway.

Reply:
Just for informational purposes: the cross-activity rules are what we've sometimes called "universal".  That's basically everything but AutoML, Regex, and Plagiarism.  So Spelling, Opinion, all the Grammar stuff, that's all cross-activity.